Editor/Analytics/GoogleTracker.cs

```diff
@@ -28,10 +31,22 @@ private string BuildEventUri(AnalyticsEvent data)
             return string.Format(uri, data.Category, data.Action, data.Label);
         }
 
+        public override void SendSessionStart()
```

Comment:
docs

Reply:
It's still an internal class. :)

Reply:
dat and the function does very pretty much says what has to be done.